### PR TITLE
 New behaviour for bottom toolbar buttons 

### PR DESCRIPTION
**Pull request by deeppandya**
Fixes : #1707 #1716

### CODE REVIEW

File: chrome/android/java/src/org/chromium/chrome/browser/ChromeActivity.java [hunk 1]
```diff
@@ -1658,6 +1661,15 @@ private void maybeRemoveWindowBackground() {
 
     @Override
     public void finishNativeInitialization() {
+
+        if(!PackageUtils.isFirstInstall(this)) {
```
Comment (deeppandya) [inline chrome/android/java/src/org/chromium/chrome/browser/ChromeActivity.java]: We should remove it in later release as it checks for users in 1.1.0 only and after updating the app to 1.2.0 we don't need it anymore.

File: chrome/android/java/res/xml/closing_tabs_preferences.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2015 The Chromium Authors. All rights reserved.
```
Comment (samartnik) [inline chrome/android/java/res/xml/closing_tabs_preferences.xml]: If it is 100% our file, not Chromium's, we should use `2019 The Brave Authors`.

File: chrome/android/java/src/org/chromium/chrome/browser/partnercustomizations/ClosingTabsManager.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
```
Comment (samartnik) [inline chrome/android/java/src/org/chromium/chrome/browser/partnercustomizations/ClosingTabsManager.java]: If it is 100% our file, not Chromium's, we should use `2019 The Brave Authors`.

File: chrome/android/java/src/org/chromium/chrome/browser/preferences/ClosingTabsPreferences.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
+package org.chromium.chrome.browser.preferences;
```
Comment (samartnik) [inline chrome/android/java/src/org/chromium/chrome/browser/preferences/ClosingTabsPreferences.java]: If it's our new file, we should add Copyright header. It's not like really important, but just for consistency.

**Comment by deeppandya** [comment]
Closing this PR as creating another one https://github.com/brave/browser-android-tabs/pull/1963

**Pull request closed without merging**